### DISCUSSION TITLE
How to configure algolia to show nesting in the search results?

### DISCUSSION THREAD

**Discussion by B4nan** [facebook/docusaurus#8669] [Q&A]
We have a lot of content in our documentation portal - https://docs.apify.com - and when we search for it, you usually end up with results that tell you only the deepest level of where the link goes, but you have no idea which "subsystem" it is. 

For example - search for `building a query`, and one of the results is for this section: https://docs.apify.com/academy/api-scraping/graphql-scraping/introspection#building-a-query

<img width="552" alt="image" src="https://user-images.githubusercontent.com/615580/219029263-52165020-261a-4af1-a04d-0dc452aa7ecd.png">

Here I can see it points to a section matching the query, and that it is in the GraphQL guide, but we would like to keep the full breadcrumbs there, so it's clear where this page actually resides. 

<img width="661" alt="image" src="https://user-images.githubusercontent.com/615580/219029529-e1a4e00b-2688-4c3b-addb-81ad1e1b24b6.png">

I guess this will be about the algolia crawler config, we use pretty much the default template for docusaurus 2, and I can see it only fills the level 0 in the result. 

Before I dive deeper, I wanted to ask if someone has a working setup for something similar?

**Comment by slorber** [comment]
Is this the thing you'd like?

![CleanShot 2023-02-15 at 19 54 07@2x](https://user-images.githubusercontent.com/749374/219125639-4ec54b50-771a-4a9a-b975-8de729d0ee25.png)


I don't know if it's possible, but that would be an interesting improvement.
Sometimes I also think our official docs search results lack a bit of context that would be helpful.

@shortcuts any idea?

**Reply by B4nan** [reply]
Kind of, I think it would better fit into the "heading" (`Markdown Features` here), as that is the "parent page" and I'd like to show its parent pages too, up to the root.

edit: maybe your idea is better, both are fine, as long as the information is somewhere
